pkgs/applications/networking/ps2client/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2018-10-18";
+  name = "ps2client-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ps2dev";
+    repo  = "ps2client";
+    rev = "92fcaf18aabf74daaed40bd50d428cce326a87c0";
+    sha256 = "1rlmns44pxm6dkh6d3cz9sw8v7pvi53r7r5r3kgwdzkhixjj0cdg";
+  };
+
```

Comment:
```suggestion
patchPhase = ''
   sed -i -e 's@-I/usr/include -I/usr/local/include@@' Makefile
'';
```

Reply:
i don't think so. Useless include path doesn't affect build , patchPhase increases complexity and update of the derivation.

Reply:
How could it affect the complexity and update? It just edits the Makefile.

And it can affect builds in non-NixOS systems which put things in those dirs.

Reply:
non-NixOS use case oki.